torch/csrc/distributed/c10d/SymmetricMemory.cpp

```diff
@@ -212,15 +213,19 @@ namespace {
 
 at::Tensor one_shot_all_reduce_meta(
     const at::Tensor& input,
+    // NOLINTNEXTLINE(performance-unnecessary-value-param)
     std::string reduce_op,
+    // NOLINTNEXTLINE(performance-unnecessary-value-param)
     std::string group_name) {
   return at::empty_like(input);
 }
 
 at::Tensor one_shot_all_reduce_copy_meta(
     const at::Tensor& symm_buffer,
     const at::Tensor& local_input,
+    // NOLINTNEXTLINE(performance-unnecessary-value-param)
     std::string reduce_op,
+    // NOLINTNEXTLINE(performance-unnecessary-value-param)
```

Comment:
nit: would it be clearer to do `(void) reduce_op` in the meta kernel? its not a big deal though

Reply:
I think dispatcher would complain about the signature mismatch.

Reply:
no, you keep the signature.

in theory this is the preferred way to tell the compiler/linter that you're intentionally not using a provided value and iiuc it should silence linter errors, but .. your way is fine too
```
void foo(int x){
   (void)x;
   return;
}
```

Reply:
Ah I see what you mean now. Thanks for the suggestion!
In this specific case, the linter wants me to change `std::string` to `const std::string&` to avoid a copy in argument passing. But `const std::string&` is unfortunately not supported by torch dispatcher signature definition (if I am not mistaken).